packages/api-client/src/importTables.js

```diff
@@ -1,11 +1,38 @@
 // @flow
 import * as R from 'ramda';
+import errorCodes from '@8base/error-codes';
 import type { Schema } from '@8base/utils';
 import type { DocumentNode } from 'graphql';
 
 import { TABLES_LIST_QUERY, TABLE_CREATE_MUTATION, FIELD_CREATE_MUTATION } from './constants';
 
-export const importTables = async (request: (query: string | DocumentNode, variables?: Object) => Promise<Object>, schema: Schema) => {
+type ImportTablesOptions = {
+  debug?: boolean,
+};
+
+const handleError = (message: string, e: Object, debug: boolean) => {
+  // eslint-disable-next-line no-console
+  console.warn(message);
```

Comment:
@zouxuoz why don't you use `console.error`?

Reply:
@vorobeez these errors don't stop the import process and just inform the user about problems in the schema